### ISSUE TITLE
Apple Carplay Support 

### ISSUE THREAD

**Issue by F3000** [remuslazar/EVPlugFinderPublic#85]
Wenn die API's von Apple für CarPlay veröffentlicht wurden. Immer mehr AUtos haben das mittlerweile , da wäre es schön es direkt auf dem Biildschirm zu haben

**Comment by remuslazar** [comment]
Ja, das ist sicher etwas, was in den kommenden Versionen eine Rolle spielen wird

**Comment by F3000** [comment]
Gibt es hier schon Neuigkeiten zu ?

**Comment by Wordcrafts** [comment]
Nein, bisher nicht. Wir arbeiten gerade an anderen Dingen und sind auch sowieso noch nicht groß genug, um bei Apple betteln gehen zu können :(

Aber: Haben wollen wir das alle. Lieber gestern als heute. :)

**Comment by F3000** [comment]
Verstehe . Ach apple sucht nur bestimmte Apps aus die dürfen :( 
Wusste ich nicht . Dachte jeder Entwickler kann ein App für CarPlay veröffentlichen. 

Was habt ihr für andere Dinge geplant ? ;)

**Comment by Wordcrafts** [comment]
Nein, dann gäbe es schon Smurfs und andere Sachen auf CarPlay ;)
Apple ist da extrem wählerisch – beschränkt es aktuell eigentlich auf "Medien".
Also Musik / Podcasts.

**Comment by F3000** [comment]
Ab heut eist es nicht mehr beschränkt. 

Wurde eben auf der WWDC 18 auch auf Apps aus dem Bereich Navigation geöffnet 👍

**Comment by F3000** [comment]
Gibt es hierzu schon was neues ?

**Comment by Wordcrafts** [comment]
Ja. Ist in Arbeit.
Da es aber keinen Sinn macht, das Karten-Interface einfach so auf einen CarPlay-Screen zu werfen (Auflösung, Tap-Genauigkeit, …) kommt das zusammen mit einem anderen Feature, an dem wir arbeiten. Also: Dauert noch ein wenig, dann kommt's aber richtig cool! :)

Nenn es einen speziellen CarPlay-Modus, mit eigenen Funktionen.

**Comment by F3000** [comment]
Ui das klingt spannend . Auf welche Funktionen darf ich mich freuen ? :) 

Gibt es vielleicht einen kleinen sneak peak ?

**Comment by Wordcrafts** [comment]
Keine weiteren Kommentare ;)
Kommt bei Zeiten, wenn die ersten Prototypen laufen und sich abfotografieren lassen.
Schau auf Twitter "@EV_Freaks" ;)

**Comment by F3000** [comment]
Ok :) 

hab mal geschaut finde aber nix passendes . Vielleicht magst du mir noch ein Twitter Link zukommen lassen :D

**Comment by LocutusB** [comment]
Wollte das Thema mal wieder hoch schubsen. Neustes Mitglied auf CarPlay ist die n-tv App. Da sollte doch ChargEV nicht länger fehlen ;-).

**Comment by Wordcrafts** [comment]
Danke!
Wir waren die letzte Zeit damit beschäftigt, das ganze Projekt auf anständige Beine zu stellen.
(https://twitter.com/EV_Freaks/status/1125757516829405190)

Danke für den Hinweis! :) Jetzt kommen wir wieder in Fahrt, waren aktuell eher gelähmt von Ämtern, Banken und Papierkrieg.
